ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -467,6 +479,8 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
             component.state.learningResourceTypes.length,
             0
           );
+          //the change to the multi terms is immediately reflected in the state
```

Comment:
Can you remove 'immediately'? Since `setState` is asynchronous we are doing this assertion in a callback so it's not quite immediate